intel_extension_for_transformers/neural_chat/config.py

```diff
@@ -80,7 +80,7 @@ class ModelArguments:
         },
     )
     use_fast_tokenizer: bool = field(
-        default=True,
+        default=False,

```

Comment:
only Llama use False for use_fast_tokenizer, is it ok to set it False on default.